test/eredis_tcp_SUITE.erl

```diff
@@ -262,7 +262,8 @@ t_faulty_logical_database(Config) when is_list(Config) ->
     process_flag(trap_exit, true),
     Res = eredis:start_link("127.0.0.1", ?PORT, [{database, 99999999},
                                                  {reconnect_sleep, no_reconnect}]),
-    ?assertMatch({error, {select_error, {unexpected_response, <<"-ERR DB", _/binary>>}}}, Res),
+    %% Note: The error message content has changed from Redis 3 to Redis 4
```

Comment:
.... so this test hasn't passed for a long time? Redis 4 must have been years ago...